src/main/java/com/google/devtools/build/lib/bazel/repository/RepositoryOptions.java

```diff
@@ -64,6 +66,32 @@ public class RepositoryOptions extends OptionsBase {
           """)
   public PathFragment repoContentsCache;
 
+  @Option(
+      name = "repo_contents_cache_gc_max_age",
+      defaultValue = "14d",
+      documentationCategory = OptionDocumentationCategory.BAZEL_CLIENT_OPTIONS,
+      effectTags = {OptionEffectTag.BAZEL_INTERNAL_CONFIGURATION},
+      converter = DurationConverter.class,
+      help =
+          """
+          Specifies the amount of time an entry in the repo contents cache can stay unused before \
+          it's garbage collected.
+          """)
```

Comment:
Should we make it so that `0` disables the GC? (This is the convention for the other GC flags)

Reply:
Done for both flags. I originally felt whoever wanted to disable GC could just use `9999d` for the delay / maxAge, but on the flip side those wanting really short delays/maxAges could just use `1ms`, so I changed it.